keyform/static/keyform/js/base.js

```diff
@@ -0,0 +1,44 @@
+$(document).ajaxSend(function(event, xhr, settings)
+{
+    function getCookie(name)
```

Comment:
I pulled this straight from RCRs, I believe. I will ask Shane about it tomorrow.